data/strategies/VoterProxy.json

```diff
@@ -2,5 +2,6 @@
   "$schema": "strategy",
   "name": "VoterProxy",
   "description": "This vault accepts deposits of {{tokens}} tokens obtained by supplying either aDai, aUSDC or aUSDT to the liquidity pool on Curve.fi. {{tokens}} tokens are staked in the gauge on Curve.fi to earn CRV rewards. Rewards are swapped for one of the underlying assets and resupplied to the liquidity pool to obtain more {{tokens}}.",
```

Comment:
why only `aDai, aUSDC or aUSDT`. It can be generalized to something like tokens that are part of the pool or something like that